docs/layouts/src/demos/SplitContent/Docs.js

```diff
@@ -8,7 +8,7 @@ import {
   Code,
   Spacing,
   Accordion,
-} from '@bandwidth/shared-components';
+} from '../../../../../src';
```

Comment:
Use absolute path

Reply:
I don't think it would work in this context. I could maybe look into setting up a webpack alias.